### PR TITLE
Version number & changelog prep for 1.0.15

### PR DESCRIPTION
**Pull request by iambmelt**
Prepares version numbers, changelog for common@1.0.15 & msal@1.2.1

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,7 @@
+Version 1.0.15
+------------
+- TODO
```
Comment (iambmelt) [inline changelog.txt]: @rpdome So what are we ultimately shipping?

- Fix for [msal#915](https://github.com/AzureAD/microsoft-authentication-library-for-android/issues/915)
    * Incorrect id_token returned for B2C app with multiple policies
- Fix for [msal#916](https://github.com/AzureAD/microsoft-authentication-library-for-android/issues/916)
    * WebView calls `loadUrl` multiple times over lifecycle
- Fix for [msal#921](https://github.com/AzureAD/microsoft-authentication-library-for-android/issues/921)
    * WebView displays error when connectivity lost
- Fix for [msal#924](https://github.com/AzureAD/microsoft-authentication-library-for-android/issues/904)
    * AT caching logic change for scope intersection
- TBD: Enhancement for WebView zoom control toggle**

** Should this force us to go `1.3.0`?

Comment (rpdome) [reply changelog.txt]: for ** I think we don't have to, given that this is an optional field in the config object.

Comment (iambmelt) [reply changelog.txt]: @rpdome From the below criteria (taken from [semver](https://semver.org/)), I'd still interpret `**` as an instance of `#2`

```
Given a version number MAJOR.MINOR.PATCH, increment the:
1. MAJOR version when you make incompatible API changes,
2. MINOR version when you add functionality in a backwards compatible manner, and
3. PATCH version when you make backwards compatible bug fixes.
```

Comment (rpdome) [reply changelog.txt]: I'm fine with that :D

File: common/src/main/java/com/microsoft/identity/common/internal/platform/Device.java [hunk 2]
```diff
@@ -73,7 +73,7 @@ private Device() {
         /**
          * The String representing the sdk platform version.
          */
-        public static final String PRODUCT_VERSION = "1.2.0-hf1";
+        public static final String PRODUCT_VERSION = "1.2.1";
```
Comment (iambmelt) [inline common/src/main/java/com/microsoft/identity/common/internal/platform/Device.java]: This is now going to be `1.3.0`

Comment (iambmelt) [reply common/src/main/java/com/microsoft/identity/common/internal/platform/Device.java]: Updated

File: changelog.txt [hunk 3]
```diff
@@ -1,3 +1,15 @@
+Version 1.0.15
+------------
+- Fix for msal#915
+    * Incorrect id_token returned for B2C app with multiple policies
+- Fix for msal#916
+    * WebView calls loadUrl multiple times over lifecycle
+- Fix for msal#921
+    * WebView displays error when connectivity lost
+- Fix for msal#924
```
Comment (shahzaibj) [inline changelog.txt]: This issue number does not seem correct

Comment (rpdome) [reply changelog.txt]: it should be 904

Comment (shahzaibj) [reply changelog.txt]: This should be common 793

https://github.com/AzureAD/microsoft-authentication-library-common-for-android/issues/793

**Pull request merged**